### ISSUE TITLE
is this null safety yet?

### ISSUE THREAD

**Issue by Danzo7** [maheshmnj/passwordfield-flutter-package#10]
I'm getting an error that tells me that the package is not [null safety]( https://dart.dev/go/unsound-null-safety) are you considering to upgrade?

```
- package:passwordfield
Error: Cannot run with sound null safety, because the following dependencies
don't support null safety:

For solutions, see https://dart.dev/go/unsound-null-safety

```

**Comment by maheshj01** [comment]
Hi @Danzo7, thanks for filing the issue. I have not migrated it to null safe yet. Will do it this week. Feel free to send a PR if you want to.

Thanks

**Comment by maheshj01** [comment]
The fix is in will land by end of this week https://github.com/maheshmnj/passwordfield/tree/feature/null-safety

**Comment by maheshj01** [comment]
Hi @Danzo7,  so sorry for the delay I got caught up in other work. 

The package is published now `passwordfield: ^0.1.0`with null safety and has some breaking changes please check the changelog and readme https://pub.dev/packages/passwordfield

Closing this issue as fixed in  https://github.com/maheshmnj/passwordfield/commit/3d981c9dc460ad8e7cee933b56f2976553dcedf5. 

Thanks for your contribution without this issue, It wouldn't have been possible.
